### DISCUSSION TITLE
Is there a way to change php fpm port?

### DISCUSSION THREAD

**Discussion by Hadatko** [docker-library/wordpress#865] [General]
Hi, sorry for bothering but cannot find the answer. Is there a way to change fpm port 9000 to something else?

**Comment by maiorano84** [comment]
A Google search for "docker php-fpm change port" led to this: https://stackoverflow.com/questions/35016435/docker-multiple-php7-fpm-containers-on-different-ports

This in turn references this thread here with some valuable information on the matter: https://github.com/docker-library/php/issues/479

**Reply by Hadatko** [reply]
Hi, thank you for reply. Last comment with "sed" made it. I am again on this task ;)
